src/rpc/misc.cpp

```diff
@@ -3,6 +3,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include "addressindex.h"
```

Comment:
I wonder if this include should go under a conditionally compiled section since it is used by the Address Indexing only.

```suggestion
#ifdef ENABLE_ADDRESS_INDEXING
#include "addressindex.h"
#endif // ENABLE_ADDRESS_INDEXING
```

Reply:
Done